doc/sections/2-technologies.tex

```diff
@@ -1,36 +1,62 @@
             time difference of arrival (TDoA), or propagation model generated from RSSI value.
-      \item \textbf{Range-free}: one of the most widely used range-free method is fingerprint localization method. This method can be categorized into three types:
-            visual fingerprint-based localization, motion fingerprint-based systems, and signal fingerprint-based methods. In our context the latter is the most promising.
+      \item \textbf{Range-free}: one of the most widely used range-free method is fingerprint localization method.
+            This method can be categorized into three types:
+            visual fingerprint-based localization, motion fingerprint-based systems, and signal fingerprint-based methods.
+            In our context the latter is the most promising.
 \end{itemize}
 
 \subsubsection{Signal fingerprint-based localization}
-Signal fingerprint-based localization is widely used in places where a large number of Wi-Fi infrastructures are deployed. This methods commonly consist of offline training phase
-and online fingerprint matching phase. The goal of the first phase is to form a fingerprint database which stores the correlation between \textit{Received Signal Strength} (RSS) from
-various \textit{Access Points}(APs) and fix locations. The device's location is determined at the matching stage. In this process we use matching algorithm to search the fingerprint
-in the database which has the minimum difference with the device that needs to be located. The associated label is our estimated location.
+Signal fingerprint-based localization is widely used in places where a large number of Wi-Fi infrastructures are deployed.
+This methods commonly consist of offline training phase and online fingerprint matching phase.
+The goal of the first phase is to form a fingerprint database which stores the correlation between \textit{Received Signal Strength} (RSS) from
+various \textit{Access Points}(APs) and fixed locations.
+The device's location is determined at the matching stage.
+In this process we use matching algorithm to search the fingerprint
+in the database which has the minimum difference with the device that needs to be located.
+The associated label is our estimated location.
+
+\subsubsection{Signal fingerprint-based localization improvement with NFC technology}
+In terms of effect, signal fingerprint-based localization has the ability to get fine-grained results.
+However, as any radio environment is dynamic: unpredictable movements of the people or large unforeseen gatherings,
+alterations in the radio network itself, and environmental effects such as change in humidity levels etc~\cite{chaudhry2013indoor}.
+Therefore, the RF values measured for location estimation at any given point in time may significantly deviate from
+those stored in the database created at the training phase.
```

Comment:
"may significantly deviate from those stored in the database" this sentence is referred to the possible skew of RF described above or is an isolated concept? If it's the last case would be better a reference supporting that statement.